{{cookiecutter.github_project_name}}/package.json

```diff
@@ -49,14 +48,14 @@
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1.10 || ^2.0.0 || ^3.0.0 || ^4.0.0"
+    "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4 || ^5 || ^6"
   },
   "devDependencies": {
     "@babel/core": "^7.5.0",
     "@babel/preset-env": "^7.5.0",
     "@jupyterlab/builder": "^3.0.0",
```

Comment:
I couldn't get the same error as you, but I did get an error about needing to increase this version:
```sh
$ jupyter labextension build --development True .
Building extension in .
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:56: UserWarning: An error occurred.
  warnings.warn("An error occurred.")
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:57: UserWarning: ValueError: Extensions require a devDependency on @jupyterlab/builder@^4.0.2, you have a dependency on 3.6.6
  warnings.warn(msg[-1].strip())
/home/bender/.local/lib/python3.10/site-packages/jupyterlab/debuglog.py:58: UserWarning: See the log file for details: /tmp/jupyterlab-debug-ji273yqd.log
  warnings.warn(f"See the log file for details: {log_path!s}")
error Command failed with exit code 1.
```

I changed the version of `@jupyterlab/builder` and was able to build then: `    "@jupyterlab/builder": "^4.0.2",`